src/ctia/http/routes/indicator.clj

```diff
@@ -117,6 +91,65 @@
       :path-params [title :- s/Str]
       :header-params [api_key :- (s/maybe s/Str)]
       :capabilities #{:list-indicators-by-title :admin}
-
       (paginated-ok
-       (list-indicators @indicator-store {:title title} params)))))
+       (list-indicators @indicator-store {:title title} params))))
+  (GET "/judgement/:id/indicators" []
+    :tags ["Indicator"]
+    :return (s/maybe [StoredIndicator])
+    :summary "Gets all indicators referencing some judgement"
+    :query [params IndicatorsListQueryParams]
+    :path-params [id :- s/Str]
+    :header-params [api_key :- (s/maybe s/Str)]
+    :capabilities #{:list-indicators-by-title :admin}
+    (paginated-ok
+     (list-indicators @indicator-store
+                      {:judgements #{{:judgement_id id}}}
```

Comment:
This id would need to be a long-ID (a URL), but here it is a short-ID from the URL.  We need to either pass in a CtiaID instance instead of the reference map, or else we need to convert the ID into a long-ID.  Code for either approach can be found in ctia.domain.id.